testing/kconfig/APKBUILD

```diff
@@ -8,9 +8,10 @@ arch="all"
 url="https://community.kde.org/Frameworks"
 license="LGPL-2.0-or-later AND LGPL-2.0-only AND LGPL-2.1-or-later"
 makedepends="extra-cmake-modules qt5-qttools-dev doxygen"
+checkdepends="xvfb-run"
 source="https://download.kde.org/stable/frameworks/${pkgver%.*}/$pkgname-$pkgver.tar.xz"
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang"
-options="!check" # Fails due to requiring running X11
+options="!check" # 2 Tests fail
```

Comment:
You can disable the specific failing tests by adding `-E "(<testname1>|<testname2>)"` to the end of the ctest command.